data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -68,10 +68,62 @@ local whole_disk_and_boot_unattended() = {
     device: 'boot-disk',
   },
 };
+
+local raid() = {
+  drives: [
+    {
+      partitions: [
+        {
+          search: '*',
+          delete: true
+        },
+        {
+          id: 'bios_boot',
```

Comment:
bios-boot

Reply:
it's bios_boot.

Reply:
it is just a label, we can name it as we want, we need to be consistent, for the other we use `-`.